AlastriaID/VerifiableAttestation/2022-11/README.md

```diff
@@ -0,0 +1,422 @@
+    </td>
+    <td>string(URI)</td>
+    <td>
+    </td>
+  </tr>
+
+  <tr>
+    <td>type</td>
+    <td>
+      Defines the credential schema type. 
+    </td>
+    <td>string</td>
+    <td>
+      MUST be FullJsonSchemaValidator2021 .
+    </td>
+  </tr>
+</table>
+
+<br></br>
+
+## Evidence 
+All evidence objects MUST contain properties id and type. Other properties are defined in data models of use cases, e.g., the presence of physical documents.
+
+<table>
+  <tr>
+    <th>Terminology</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Notes</th>
+  </tr>
+
+  <tr>
+    <td>id</td>
+    <td>
+      References a record of evidence. 
+    </td>
+    <td>string(URI)</td>
+    <td>
+    </td>
+  </tr>
+
+  <tr>
+    <td>type</td>
+    <td>
+      Defines the evidence type. 
+    </td>
+    <td>string</td>
+    <td>
+    </td>
+  </tr>
+</table>
+
+## Proof 
+
+<s><table>
+  <tr>
+    <th>Terminology</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Notes</th>
+  </tr>
+
+  
+
+  <tr>
+    <td>type</td>
+    <td>
+      Defines the proof type.	 
+    </td>
+    <td>string</td>
+    <td>
+    </td>
+  </tr>
+
+<tr>
+    <td>created</td>
+    <td>
+      Defines the date and time, when the proof has been created.	 
+    </td>
+    <td>dateTime</td>
+    <td>
+    </td>
+  </tr>
+
+<tr>
+    <td>proofPurpose</td>
+    <td>
+      Defines the purpose of the proof.		 
+    </td>
+    <td>string</td>
+    <td>
+    </td>
+  </tr>
+
+
+<tr>
+    <td>verificationMethod</td>
+    <td>
+      Contains information about the verification method/proof mechanisms..
+    </td>
+    <td>string(URI)</td>
+    <td>
+    </td>
+  </tr>
+
+
+<tr>
+    <td>jws</td>
+    <td>
+      Defines the proof value in JWS signature format.	 
+    </td>
+    <td>string</td>
+    <td>
+    </td>
+  </tr>
+</table></s>
+
+<br><br>
+
+## Terms of use 
+<s>Mandatory for Accredited Verifiable Attestations.<br>
+<b>MUST be equal to VerifiableAccreditation.</b></s>
+
+<s><table>
+  <tr>
+    <th>Terminology</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Notes</th>
+  </tr>
+
+  
+
+  <tr>
+    <td>id</td>
+    <td>
+      References a record of terms of use.	 
+    </td>
+    <td>string(URI)</td>
+    <td>
+      <s>If used for Verifiable Accreditation as in an Accredited Verifiable Attestation: References the Verifiable Accreditation stored as an attribute of the issuer in the Trusted Issuer Registry (TIR).</s> 
+    </td>
+  </tr>
+
+<tr>
+    <td>type</td>
+    <td>
+      Defines the term of use type.	 
+    </td>
+    <td>string</td>
+    <td>
+      <s>If used for Verifiable Accreditation as in an Accredited Verifiable Attestation: MUST be equal to a VerifiableAccreditation type.</s> 
+    </td>
+  </tr>
+</table></s>
+
+<br>
+
+All the fields are required except validUntil, expirationDate, credentialStatus, evidence, proof and termsOfUse.<br>
```

Comment:
This point is important, maybe I would add a column in the table to indicate if the field is mandatory or not.

Do you think it is a good idea?